app/controllers/users_controller.rb

```diff
@@ -1,9 +1,10 @@
 class UsersController < ApplicationController
   def index
-    @users = User.all
+    @users_all = User.all
   end
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/231341631-46d4284e-aeff-4acf-86cd-0bb3ebcf6c8f.png)

- [ ] When I tried to create a comment then I saw this error in the terminal :fearful: Kindly fix this error to make your project follow the requirements :+1: